config/model_config.json

```diff
@@ -4,56 +4,34 @@
             "ML_MODEL_CONFIG_NAME": "london_taxi",
             "ENV_NAME": "dev",
             "AZURE_RM_SVC_CONNECTION": "azure_connection",
```

Comment:
Is this needed here? We are obtaining this one from the Variable Group

Reply:
+1